### PR TITLE
[PM-12316] Implement inline menu passkeys authenticating state

### PR DESCRIPTION
**Pull request by cagonzalezcs**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-12316

## 📔 Objective

When a user selects a passkey from the inline menu, the act of authenticating is fairly abrupt. We’d like to improve that by including a “loading state” for the user indicating that an authentication is occurring.

## 📸 Demo

https://github.com/user-attachments/assets/d1167b14-2081-4a5d-87f4-e2f98c5bcba1

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by danielleflinn — DISMISSED** [review]
screen recording looks good

File: apps/browser/src/autofill/utils/index.ts [hunk 1]
```diff
@@ -426,3 +426,37 @@ export function getSubmitButtonKeywordsSet(element: HTMLElement): Set<string> {
...
+/**
+ * Generates the origin and subdomain match patterns for the URL.
+ *
+ * @param url - The URL of the tab
+ */
+export function generateDomainMatchPatterns(url: string): string[] {
+  try {
+    if (!url.startsWith("http")) {
+      url = `https://${url}`;
+    }
```
Comment (jprusik) [inline apps/browser/src/autofill/utils/index.ts:439]: Is there a scenario where `url` will start with a different protocol? If not, then are we able to skip this check? If so, it seems like origin will be parsed incorrectly during `originMatchPattern` assignment.

e.g.
Screenshot 2024-09-19 at 1 01 58 PM

Comment (cagonzalezcs) [reply apps/browser/src/autofill/utils/index.ts:439]: Good point... hmm... maybe a regex checking for `*://*` would work better here instead...

What do you think?

**Comment by jprusik** [comment]
BIT run:
https://github.com/bitwarden/browser-interactions-testing/actions/runs/10948807410

**Review by jprusik — DISMISSED** [review]
We should probably come back to `generateDomainMatchPatterns` at some point, but LGTM for now

**Review by jprusik — APPROVED** [review]

**Pull request merged**